### DISCUSSION TITLE
Timeout after x minutes

### DISCUSSION THREAD

**Discussion by Mijinion** [jagrosh/MusicBot#1146] [Questions] [answered]
Is there a way to make it so the bot disconnects after a set amount of time? I know it can disconnect after each queue but I find that really annoying.

**Comment by YouKaih** [comment, marked as answer]
Hi! you enable stay in channel in config.txt file :) It's on the 91st line. You can make it stay in the vc with the 105th line if you set it on 0 :) Hope this help a little

**Reply by MichailiK** [reply]
For now, enabling `stayinchannel` and setting `alonetimeuntilstop` is indeed the best solution there currently is.
